Scripts/README.md

```diff
@@ -0,0 +1,3 @@
+# Scripts
+
+Scripts for MemoArt Continuous Integration
```

Comment:
Could be nice to add how to use this script and explicitly mention that for now, only one script is available: "Marketing Screenshots generation"

Reply:
Done